src/TagHelperPack/AuthzTagHelper.cs

```diff
@@ -12,13 +12,17 @@ namespace TagHelperPack;
 /// </summary>
 [HtmlTargetElement("*", Attributes = AspAuthzAttributeName)]
 [HtmlTargetElement("*", Attributes = AspAuthzPolicyAttributeName)]
+[HtmlTargetElement("*", Attributes = AspAuthzRoleAttributeName)]
+[HtmlTargetElement("*", Attributes = AspAuthzPolicyPermissionAttributeName)]
 public class AuthzTagHelper : TagHelper
 {
     internal static object SuppressedKey = new();
     internal static object SuppressedValue = new();
 
     private const string AspAuthzAttributeName = "asp-authz";
     private const string AspAuthzPolicyAttributeName = "asp-authz-policy";
+    private const string AspAuthzRoleAttributeName = "asp-authz-role";
+    private const string AspAuthzPolicyPermissionAttributeName = "asp-authz-permission";
```

Comment:
I'm not super comfortable with this name as it's too specific to the authorization handler implementation in the sample. The "permission" isn't a strongly-typed primitive/concept of the authorization system but instead is passed through the `object resource` parameter which is intended as an untyped escape-hatch to pass arbitrary information from the authorization site to the authorization handler. Perhaps your change could be updated to reflect that, e.g. change the name to `asp-authz-resource` and type it as `object`.